ee/tabby-db/migrations/08-repositories-table/up.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE repositories (
+    id INTEGER PRIMARY KEY AUTOINCREMENT,
+    name VARCHAR(255) NOT NULL,
+    git_url VARCHAR(255) NOT NULL,
```

Comment:
let's still add `created_at` just for sorting.

Reply:
If it's just for sorting, wouldn't sorting by ID still order it the same?